__tests__/index.test.ts

```diff
@@ -678,6 +722,32 @@ describe('Auth0', () => {
           max_age: undefined
         });
       });
+
+      it('calls `tokenVerifier.verify` with the organization ID from in the transaction, if set', async () => {
+        const { auth0, tokenVerifier, transactionManager } = await localSetup();
+
+        transactionManager.get.mockReturnValue({
+          code_verifier: TEST_RANDOM_STRING,
+          nonce: TEST_RANDOM_STRING,
+          audience: 'default',
+          scope: TEST_SCOPES,
+          redirect_uri: 'http://localhost',
+          organizationId: TEST_ORG_ID
+        });
+
+        await auth0.handleRedirectCallback();
+
+        expect(tokenVerifier).toHaveBeenCalledWith({
+          id_token: TEST_ID_TOKEN,
+          nonce: TEST_RANDOM_STRING,
+          aud: 'test-client-id',
+          iss: 'https://test.auth0.com/',
+          leeway: undefined,
+          max_age: undefined,
+          organizationId: TEST_ORG_ID
+        });
+      });
```

Comment:
Do we want to use this test or move to a less mock approach ? We could combine the tests added to index.test.
ts file into a single one and avoid mocking the transactionManager.

Reply:
Yeah I think we will do that, as we're going through a round of test refactoring anyway I figured we could do that as part of that work as I don't think this section has been moved yet (correct me if I'm wrong).

Reply:
No that is correct.